### ISSUE TITLE
Parsing fails if test case result is empty

### ISSUE THREAD

**Issue by ulfsauer0815** [EnricoMi/publish-unit-test-result-action#24]
Parsing fails when the test case result is missing attributes or content

A Java JUnit 5 test generates the following XML test case, which fails parsing and breaks publishing.

```xml
<testcase name="test" classname="test.Test" time="0.042">
  <skipped/>
</testcase>
```
fails with the message
```
TypeError: argument of type 'NoneType' is not iterable
```

The culprid seems to be missing null checks here:
https://github.com/EnricoMi/publish-unit-test-result-action/blob/55a357a9b2d6aba0a2613137aa5eaf7d7cb714e6/publish_unit_test_results.py#L46-L47

**Comment by EnricoMi** [comment]
Thanks for raising this new case, I will fix it asap.

**Comment by EnricoMi** [comment]
@ulfsauer0815 can you try this branch with your test result files, please?

        - uses: EnricoMi/publish-unit-test-result-action@branch-work-with-no-test-case-attributes

**Comment by ulfsauer0815** [comment]
Works like a charm!
Thank you so much! That was an incredibly fast reply and fix! :)

**Comment by EnricoMi** [comment]
Released as v1.3, thanks for the detailed bug report.
